### DISCUSSION TITLE
How much libinput configuration is wanted ?

### DISCUSSION THREAD

**Discussion by PeterOGB** [WayfireWM/wayfire#1968] [General]
If I understand it correctly under wayland it is the compositors job to configure libinput rather than using files like 
/usr/share/X11/xorg.conf.d/40-libinput.conf  which exposed all of the libinput configuation options.

I've been reading the libinput documentation and it seems there are more things that could usefully be configured than are currently catered for in the existing options in the [input] section of wayfire.ini.  

How much more configuration flexibility would be helpful / welcomed ?

PeterO

**Comment by ammen99** [comment]
So far, it has been like this: when someone wants a particular functionality, they can send a PR for it. This way, each option is guaranteed to have at least one user :)

**Reply by PeterOGB** [reply]
Ok, I'll stick to getting the button scrolling sorted for now as that will have at least one user :-)
I'm working a matrix of device types vs their features  to try and avoid any duplication of options but also making sure options are only applied where they are appropriate, but this may need a way to optionally identify the device in the same way libinput.conf files did, but that's something for later discussions.
